src/main/java/org/cryptacular/bean/EncodingHashBean.java

```diff
@@ -64,9 +71,44 @@ public void setCodecSpec(final Spec<Codec> codecSpec)
   }
 
 
+  /**
+   * Whether data provided to {@link #hash(Object...)} includes a salt as the
+   * last parameter.
+   *
+   * @return  whether hash data includes a salt
+   */
+  public boolean isSalted()
+  {
+    return salted;
+  }
+
+
+  /**
+   * Sets whether {@link #hash(Object...)} should expect a salt as the last
+   * parameter.
+   *
+   * @param  salted  whether hash data includes a salt
+   */
+  public void setSalted(final boolean salted)
+  {
+    this.salted = salted;
+  }
+
+
   @Override
```

Comment:
I think this method could benefit from a javadoc comment that explains that the output is a function of the `salted` flag. This is the only `hash` method that can produce anything other than the hash bytes.